cognite/powerops/resync/config/production/plant.py

```diff
@@ -26,6 +26,7 @@ class Plant(BaseModel):
     p_max: float = p_max_fallback  # arbitrary max power output if not specified
     head_loss_factor: float = 0.0
     penstock_head_loss_factors: Optional[dict[str, float]] = None
+    connections_losses: Optional[float] = None
```

Comment:
Øivind asked if we could include potential conneciton losses in plant metadata so they could differentiate the plant that have and do not have connection losses